libs/ui/src/diagnostics/mark_scan_readiness_report.test.tsx

```diff
@@ -41,7 +48,9 @@ test('MarkScanReadinessReport', () => {
   screen.getByText(/All Precincts/);
   screen.getByText('Battery Level: 50%');
   screen.getByText('Power Source: Battery');
-  screen.getByText('Detected');
-  screen.getByText(/Test passed/);
+  screen.getByTestId('paperHandlerDetected');
+  screen.getByTestId('accessibleControllerDetected');
+  screen.getByTestId('paperHandlerTestPassed');
+  screen.getByTestId('accessibleControllerTestPassed');
```

Comment:
A pattern I've used in these cases is to find the section title, get its container using `.closest('section')` or similar, and then use `within` to find the text I'm looking for within that container. A bit more robust than using a test id since you still test the actual content displayed